rktlet/runtime/pod_sandbox.go

```diff
@@ -89,7 +89,7 @@ func (r *RktRuntime) RunPodSandbox(ctx context.Context, req *runtimeApi.RunPodSa
 		if err == nil {
 			break
 		}
-		if status.GetStatus().State != runtimeApi.PodSandboxState_SANDBOX_READY {
+		if status.GetStatus() == nil || status.GetStatus().State != runtimeApi.PodSandboxState_SANDBOX_READY {
```

Comment:
should we avoid to call `GetStatus()` twice by using a variable like you did before?

Reply:
Somehow I missed this comment

Reply:
Updated.